internal/infrastructure/kubernetes/ratelimit/resource.go

```diff
@@ -122,6 +135,18 @@ func expectedRateLimitContainerEnv(ratelimit *egcfgv1a1.RateLimit, rateLimitDepl
 			Name:  UseStatsdEnvVar,
 			Value: "false",
 		},
+		{
+			Name:  ConfigType,
+			Value: "GRPC_XDS_SOTW",
```

Comment:
for now it passes the test using cases in https://gateway.envoyproxy.io/v0.4.0/user/rate-limit.html, but still need to test `ConfigType: FILE` mode out, in case someone is tampering with this variable

Reply:
if so, do we need to prevent someone is changing the `ConfigType` to `FILE`, maybe add some validations?

Reply:
maybe the user want want to override to `FILE` manually, its fine to ignore that case imo